client/components/Nav.jsx

```diff
@@ -91,7 +91,10 @@ class Nav extends React.PureComponent {
           })()}
           <li className="nav__item">
             <p className="nav__open">
-              <Link to="/p5/sketches">
+              <Link
+                to="/p5/sketches"
+                onClick={this.props.stopSketch}
```

Comment:
Oh that's right - there may be some extra `stopSketch` calls in the code, I'll go through and clean them up